wp-content/themes/twentyseventeen-wp20/style.css

```diff
@@ -419,53 +423,231 @@ body.page .entry-header {
+}
+
+.wp15-locale-switcher-container .select2-container .select2-selection--single .select2-selection__rendered {
+	padding-right: 22px !important;
+}
+
+.wp15-locale-switcher-container.select2-selection--single .select2-selection__rendered {
+	color: var(--wp20--color--grey-6) !important;
+}
+
+@media screen and (min-width: 48em) {
+	.wp15-locale-switcher-container .select2-container--focus .select2-selection__rendered {
+		outline: thin dotted;
+	}
+}
+
+/* Arrow indicators */
+.wp15-locale-switcher-container .select2-selection--single .select2-selection__arrow b {
+	width: 5px !important;
+	height: 5px !important;
+	border: 1px solid var(--wp20--color--grey-0) !important;
+	border-bottom: none !important;
+	border-left: none !important;
+	transform: rotate(135deg) !important;
+}
+
+.wp15-locale-switcher-container .select2-container--open .select2-selection--single .select2-selection__arrow b {
+	transform: rotate(-45deg) !important;
+}
+
+@media screen and (min-width: 48em) {
+	.wp15-locale-switcher-container {
+		position: absolute;
+		top: 0;
+		right: 0;
+	}
+}
+
+/*
+ * Locale Switcher: Dropdown portion of Select2
+ */
+
+.navigation-top-menu-container .select2-container--default .select2-selection--single {
+	text-align: left;
+	border-bottom: 1px solid var(--wp20--color--border);
+}
+
+.navigation-top-menu-container .select2-container--default,
+.navigation-top-menu-container .select2-container .select2-dropdown {
+	text-align: right;
+}
+
+.navigation-top-menu-container .select2-search--dropdown {
+	padding: 0;
+}
+
+.navigation-top-menu-container  .select2-search--dropdown .select2-search__field {
+	padding: 12px 40px 12px 12px;
+	border-radius: 0;
+	border: 0 !important;
+	border-bottom: 1px solid var(--wp20--color--border) !important;
+	color: var(--wp20--color--text);
+	background-image:  url( 'images/search-icon.svg' );
+	background-repeat: no-repeat;
+	background-size: 12px 12px;
+	background-position: calc(100% - 20px) center;
+	font-size: 14px;
+	text-align: right;
+}
+
+.navigation-top-menu-container  .select2-search--dropdown .select2-search__field::placeholder {
+	color: var(--wp20--color--text-meta);
+}
+
+.navigation-top-menu-container .select2-container .select2-dropdown {
+	border: 0;
+}
+
+.navigation-top-menu-container .select2-results__option {
+	font-size: 14px;
+	padding: 6px 12px;
+}
+
+@media screen and (max-width: 48em) {
+	.navigation-top-menu-container .select2-results__options {
+		border-bottom: 1px solid var(--wp20--color--border);
+	}
+}
+
+.navigation-top-menu-container .select2-container--default .select2-results__option,
+.navigation-top-menu-container .select2-container--default .select2-results__option--highlighted[aria-selected]
+.navigation-top-menu-container .select2-container--default .select2-results__option--highlighted[aria-selected=false] {
+	background-color: transparent !important;
+	color: var(--wp20--color--text) !important;
+	padding: 12px var(--wp20--spacing--edge-space);
```

Comment:
This var changes above tablet size from [20px](https://github.com/WordPress/wp20.wordpress.net/blob/trunk/wp-content/themes/twentyseventeen-wp20/style.css#L35) to [80px](https://github.com/WordPress/wp20.wordpress.net/blob/trunk/wp-content/themes/twentyseventeen-wp20/style.css#L160).

I realise that's not very obvious. Maybe it should be renamed to reference `.wrap` as that's what the primary use is.

Might want another var which just represents `20px`

Reply:
We can probably just use `20px`.